CHANGELOG.md

```diff
@@ -15,20 +15,26 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   the user isn't authorized.
 - Add two new sections in DjangoCMS courses: Accessibility and Required
   Equipment.
-- Ongoing product displayed on the syllabus without active enrollment 
+- Ongoing product displayed on the syllabus without active enrollment
   now link to the order details page in the learner dashboard.
 
 ## Changed
 
+- Update configuration setting `STATICFILES_STORAGE` and
+  `DEFAULT_FILE_STORAGE` with `STORAGES` dictionary configuration
+- Pin Django version < 5 and dropped support for MySQL version < 8
+- Upgrade to Django 4.1 version
+- Upgrade to Django 4.0 version and postgres to version 12 in
+  docker compose file
```

Comment:
IMO, it does not make sense to add an entry to each upgrade of Django. The final result of your work is that you upgrade to Django 4.2, the CHANGELOG should not reflect the fact you updated progressively.

Reply:
Okay, I will adjust this file right away. 
I thought it would be good to insert a line for each commit within this Pull Request.